model/src/main/scala/uk/gov/ons/addressIndex/model/server/response/AddressResponse.scala

```diff
@@ -185,7 +185,9 @@ object AddressResponseAddress {
     val geo: Try[AddressResponseGeo] = for {
```

Comment:
I think the naming of these classes should be revisited, consider namespacing instead of adding another prefix.